### DISCUSSION TITLE
Too many redirects

### DISCUSSION THREAD

**Discussion by TwoBrake** [outline/outline#4964] [Self Hosting]
Hey there, I am having an issue after I setup Outline on my Ubuntu server. When I perform the following steps provided below, it constantly redirects me to the same URL. I don't know why, please let me know if there is any fix.

**To Reproduce**
1. Start your instance with `yarn start`.
2. Go to the website you setup with Nginx reverse proxy and an SSL provided by Let's Encrypt.
3. You will get error too many redirects and your console will be spammed.

 The console spams this:
```
{"label":"http","level":"info","message":"  \u001b[90m-->\u001b[39m \u001b[1mGET\u001b[22m \u001b[90m/\u001b[39m \u001b[36m301\u001b[39m \u001b[90m2ms\u001b[39m \u001b[90m83b\u001b[39m"}
```

Thank you in advance.

**Comment by tracure1337** [comment]
This is an issue/setting within your reverse proxy imo.
